install.php

```diff
@@ -1,3 +1,40 @@
 <?php
-$_GET["mod"] = "install";
-include_once("index.php");
+
+//raise error reporting - could be interesting and this is autmomatically reset when index.php is executed
+error_reporting(E_ALL);
+
+
+/**
+* Check for common failures
+*
+* Small function to check against a number of dependencies that cause install.php to fail without showing any error details
+* @return bool check result. True if error found, false if all OK
+*/
+function checkdeps(){
+
+	$error =false;
+	$errmsg='The following issues were found while preparing to run the installation:<br/>'. PHP_EOL;
+
+	//check if composer was executed (folder ./vendor created)
+	if (!is_dir(__DIR__ . '/vendor')) {
+
+		$error = true;
+		$errmsg .= __DIR__ . '/vendor directory is missing. Did you run composer?<br/>' .PHP_EOL;
```

Comment:
Why we put a `<br />` at the end?

Reply:
That one is fairly simple to answer:
As there is no previous/later page output, this is shown as-is.
The browser irgnores the EOL but <br/> does still enforce a line break, so that the different text pieces are shown in their own line, now matter how viewed

Reply:
Thanks! Fine for me :)